tests/test_helpers.py

```diff
@@ -69,53 +68,56 @@ def test_get_source_from_index_without_dash():
 
 def test_parse_records():
     records = list(helpers.parse_records("tests/fixtures/sample_records.json"))
-    assert len(records) == 6
+    n_sample_records = 6
+    assert len(records) == n_sample_records
```

Comment:
Not requesting a change here, but in my opinion this "magic comparison" thing linting error is probably not something we need to worry about in tests; I think it's okay to use direct number comparisons for simple logic like this.  Appreciate the effort to meet high linting requirements, but going forward I'd propose it's not needed.

Reply:
I'll keep this in mind!